packages/core-flows/src/definition/cart/workflows/add-to-cart.ts

```diff
@@ -96,6 +97,9 @@ export const addToCartWorkflow = createWorkflow(
     })
 
     refreshCartPromotionsStep({ id: input.cart.id })
+    refreshPaymentCollectionForCartStep({
```

Comment:
NIT: There is a comment that `update payment sessions` is missing, I guess we can remove that now